### DISCUSSION TITLE
Default Window Manager with Wayland update

### DISCUSSION THREAD

**Discussion by crftbt** [lxqt/lxqt#2573] [General]
With the upcoming LXQt 2.1 Wayland update, are there any plans to deprecate OpenBox (which hasn't been maintained for 9 years)?

**Comment by tsujan** [comment]
LXQt hasn't recommended Openbox for years. In fact, on several occasions, it has been mentioned that Openbox is dead and has bugs, especially with multi-screen setups. But users are free to use it; we can't "deprecate" it because it has no place in our codes.

**Comment by crftbt** [comment]
A few tests of Window Managers:

lxqt + sddm + jwm		2022.10.22 = 229MB
lxqt + sddm + pekwm		2023.01.23 = 249MB
lxqt + sddm + metacity	2024.03.20 = 265MB

lxqt + sddm + openbox	2015.07.01 = 242MB
